sdcm/cluster_aws.py

```diff
@@ -893,6 +894,7 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,  # pylint:
                  n_nodes=3,
                  params=None,
                  node_type: str = 'scylla-db',
+                 nodes_smp=[]
```

Comment:
don't initialize an argument with a mutable list
set it as None, and later in code initialize the list if needed